docs/guide/custom-ui.md

```diff
@@ -198,6 +198,49 @@ The `AposWidget` component has **nothing to do with a typical site visitor exper
 Before you override an editor modal, consider [adding a custom schema field type](/guide/custom-schema-field-types.md) instead.
 :::
 
+## Adding custom context menu items
+
+Starting from Apostrophe v3.18.0 we can add custom context menus (edit mode) from within any module, targeting any Vue component that implements `AposModal`. The menu registration should happen in the initialization phase.
```

Comment:
You don't need to specify "starting from..." it tends to accumulate and we can send a more general message to developers to update if they don't see stuff.

Reply:
(i.e. we can send that message elsewhere once and not in every page.)

Reply:
"custom context menu items," not "custom context menus." Same issue with the screenshot.